src/elements/common/create-folder-dialog/CreateFolderDialog.js

```diff
@@ -106,7 +107,9 @@ const CreateFolderDialog = ({
                     </div>
                 ) : null}
                 <FormattedMessage tagName="div" {...messages.createDialogText} />
-                <input ref={ref} onKeyDown={onKeyDown} required type="text" />
+                <div className="be-modal-input">
```

Comment:
Is this div required?

Reply:
the styling didn't apply when targeted to the input but did when i added the div.